pkgs/development/compilers/dotnet/sdk/default.nix

```diff
@@ -0,0 +1,280 @@
+{ stdenv
```

Comment:
.NET 6 is already on release 6.0.400, shouldn't we upgrade to that?

Reply:
6.0.108 is currently the latest supported for source build: https://github.com/dotnet/source-build#support

Eventually I'll look at what's required to build newer versions, but I think these supported ones will still be needed to bootstrap.

I do know that 6.0.108 will build with the binary distro of 6.0.107 or with itself as bootstrap, but not with 6.0.400.

Reply:
I see, very unfortunate they don't seem to support later versions as those have CVE fixes.